### ISSUE TITLE
Need to handle when SRS and instance record are not linked

### ISSUE THREAD

**Issue by shelleydoljack** [sul-dlss/libsys-airflow#879]
https://github.com/sul-dlss/libsys-airflow/blob/f864b18a31e3406256511fa6def5000998262f48/libsys_airflow/plugins/data_exports/marc/exports.py#L70-L87

This endpoint returns 404 if the SRS is not linked to the instance record. We need to handle this error case, especially when we are doing a full dump of data.

**Comment by shelleydoljack** [comment]
Example response body:
```
Error retrieving Record by externalId: 'e869f294-dfc3-480f-84f0-4e47c2fe5eab', response code 404, Not Found
```

**Comment by jgreben** [comment]
Fixed by #880
